src/System.Runtime/tests/System/String.cs

```diff
@@ -1176,156 +1152,126 @@ public static void TestOperators()
         Assert.True(b);
     }
 
-    [Fact]
-    public static void TestPad()
+    [Theory]
+    [InlineData("Hello", 5, ' ', "Hello")]
+    [InlineData("Hello", 7, ' ', "  Hello")]
```

Comment:
The original tests had cases for padding with something other than a space.  We shouldn't lose those tests.